infrastructure/functions/output.tf

```diff
@@ -0,0 +1,3 @@
+output "prod_resource_group" {
+  value = azurerm_resource_group.fap-backend-resource-group
+}
```

Comment:
Ich weiß nicht, was hier best practise ist, aber ein Vorschlag:
Statt die resource_group, in der alle Sachen liegen, im functions modules zu erstellen und dann rauszureichen, könnte man sie doch auch in einer der main.tfs (root oder infrastructure/main.tf) erstellen und dann reinreichen - dann löst man sie von den functions.